sjb/actions/post_script.py

```diff
@@ -12,7 +12,7 @@ class PostScriptAction(Action):
     """
 
     def __init__(self, repository, script, title):
-        self.action = ScriptAction(repository, script, title)
+        self.action = ScriptAction(repository, script, title, None)
```

Comment:
No reason this can't have a timeout right? Should thread through all the arguments